packages/schemas/src/player/gamemodes/quests/modes/murdermystery.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Statsify
+ *
+ * This source code is licensed under the GNU GPL v3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ * https://github.com/Statsify/statsify/blob/main/LICENSE
+ */
+
+import { FormattedGame } from "../../../../game";
+import { createGameModeQuests } from "../util";
+
+export const MurderMysteryQuests = createGameModeQuests({
+  game: FormattedGame.MURDER_MYSTERY,
+  fieldPrefix: "mm",
+  daily: [
+    { field: "win", propertyKey: "daily_winner" },
+    { field: "power_play", propertyKey: "daily_powerPlay" },
+    { field: "target_kill", propertyKey: "daily_hitman" },
```

Comment:
I think `field` and `propertyKey` got flip flopped here. Something doesn't seem right.